pkg/app/nginx.go

```diff
@@ -0,0 +1,62 @@
+package app
+
+import (
+	"github.com/maistra/maistra-test-tool/pkg/examples"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+type nginx struct {
+	ns   string
+	mTLS bool
+}
+
+var _ App = &nginx{}
+
+func Nginx(ns string) App {
+	return &nginx{ns: ns, mTLS: false}
+}
+
+func NginxWithMTLS(ns string) App {
+	return &nginx{ns: ns, mTLS: true}
+}
+
+func (a *nginx) Name() string {
+	return "nginx"
+}
+
+func (a *nginx) Namespace() string {
+	return a.ns
+}
+
+func (a *nginx) Install(t test.TestHelper) {
+	t.T().Helper()
+	oc.CreateGenericSecretFromFiles(t, a.Namespace(),
+		"nginx-ca-certs",
+		"example.com.crt="+examples.NginxServerCACert())
+	if a.mTLS {
+		oc.CreateTLSSecret(t, a.Namespace(), "nginx-server-certs", examples.MeshExtServerCertKey(), examples.MeshExtServerCert())
```

Comment:
Should we generate certificates in a test run or reuse the sample ones ?

Reply:
Ignore my comment above. I will try to complete a generate tls cert util tool next Monday. 
And then we can review and evaluate if we want to use sample certs or generating them in runtime.